src/main/java/by/itech/lab/supplier/dto/mapper/CustomerMapper.java

```diff
@@ -30,11 +30,17 @@ public CustomerDto map(Customer entity) {
         return CustomerDto.builder()
                 .id(entity.getId())
                 .name(entity.getName())
-                .registrationDate(entity.getRegistrationDate().toLocalDate())
+                .registrationDate(entity.getRegistrationDate())
                 .status(entity.isStatus())
                 .build();
     }
 
+    public void update(final CustomerDto from, final Customer to) {
```

Comment:
method name unclear

Reply:
How should I name it? "updateCustomer"? I suppose my name is obviously in this context.

Reply:
changed to updateCustomer, cuz create updateStatus method in CustomerMapper

Reply:
it is a data transfer object, not a repository where you can update any entity, how can you update CustomerDto? I leave it up to you ...